.github/workflows/ci.yml

```diff
@@ -113,6 +113,12 @@ jobs:
       - name: Build
         run: PRESERVE_NEXT_CACHE=true npm run build
 
+      - name: Save .next build (frontend)
+        uses: actions/cache/save@v3
+        with:
+          path: .next
+          key: ${{ runner.os }}-next-build-${{ github.sha }}
+
```

Comment:
As per https://nextjs.org/docs/pages/building-your-application/deploying/ci-build-caching, NextJS recommends caching only `.next/cache`. Is this not covered already by https://github.com/opencollective/opencollective-frontend/blob/9236a8577a86a6831329663466f7d03ddfec9851/.github/workflows/ci.yml#L104?

Reply:
I think the recommendation makes sense for isolated CI runs, here its different because the e2e step depends on the build step so we are just moving an artifact between steps in the same workflow.
If instead we just save the cache, we would spend some time rebuilding what we just did (times 4 due to the e2e matrix).

Reply:
Makes sense; maybe we could unify and have a single `.next` cache then